ixofoundation/ixo-webclient
# chore(project): hookup control panel to schema

## Scope

https://nonaredteam.atlassian.net/browse/IXO-392
https://nonaredteam.atlassian.net/browse/IXO-393
https://nonaredteam.atlassian.net/browse/IXO-394
https://nonaredteam.atlassian.net/browse/IXO-395

## Work Done

Added a control panel schema
Refactored control panel to split up components that use schema settings
Fixed a bug with the filtering defaulting to certain view
Prepared the selector to handle different entityTypes

## Steps to Test

Take a look at the schema in the control panel and verify that it looks good
View a project and see that it pulls in the schema
